.codegen/service.py.tmpl

```diff
@@ -188,10 +188,10 @@ class {{.Name}}API:{{if .Description}}
         {{template "method-call" .}}
 
     {{if and .Wait (and (not .IsCrudRead) (not (eq .SnakeName "get_run"))) }}
```

Comment:
Thanks for doing this, LGTM, I have one concern -- this might lead to confusion on when we use `.SnakeName` and when `template "safe-snake-name" .`. Should we use safe-snake-name everywhere? 

Longer term, I think we will remove this once we will have spec linter and then it would be one time effort to rename these back to `.SnakeName`

Reply:
No, you only need to use `template "safe-snake-name" .` when generating a field or method name. There are places where we check whether the operation is `get_run`. In that case, it doesn't matter too much, since that is not a reserved keyword.

With this PR in place, there isn't a need for the linter anymore, since we'll safely rename fields that conflict with keywords.